JS-files/about.js

```diff
@@ -0,0 +1,15 @@
+// About Page Navigation
+const menuBtn = document.getElementById('burger-container');
+const navOverlay = document.getElementById('nav');
+const navList = document.getElementsByClassName('menu')[0];
+const burgerBar = document.getElementsByClassName('hamRotate')[0];
+function overlay() {
+  navOverlay.classList.toggle('invisible');
+  burgerBar.classList.toggle('active');
+  navOverlay.style.animation = 'awesome-out 1s';
+}
+
+if (window.innerWidth <= 768) {
+  menuBtn.addEventListener('click', overlay);
+  navList.addEventListener('click', overlay);
+}
```

Comment:
I did this because of the animation above I don't want to apply it on desktop ;

- also I think if I apply animation none important in CSS it should be fixed 
- suggestion are welcome